package.json

```
@@ -71,7 +70,7 @@
     "@types/jest": "29.5.12",
     "@types/jest-axe": "3.5.9",
     "@types/lodash.debounce": "4.0.9",
-    "@types/marked": "5.0.2",
+    "@types/markdown-it": "^13.0.7",
```

```suggestion
    "@types/markdown-it": "13.0.7",
```

Normal que ça ne soit pas la même version que `markdown-it` ?

---

j'ai pris la derniere version donc oui